### ISSUE TITLE
create an umbraco example

### ISSUE THREAD

**Issue by jesperordrup** [KOBENDigital/bento.editor#1]
Hi Peter,

I think adaption would be fast if you add an example installation

Testing this later!! Description sounds promising.
Jesper

**Comment by pgregorynz** [comment]
Good idea and we will get something up hopefully soon.  In the meantime, If you check out the repo and build it the Bento.Website project is a working example.

**Comment by danieljameswilliams** [comment]
Hi Peter,

When I download the project, run in VS and login to Umbraco, I get an error message in the console:
![image](https://user-images.githubusercontent.com/4445229/95568763-ead72200-0a24-11eb-8689-fb0832c6c594.png)

I tried both the repo download and latest release - What am I missing?

**Comment by pgregorynz** [comment]
Hmm I have not seen this before.  I will take a look when I get home in a few hours time.
